platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/GuestClaimsHandlerExt.java

```diff
@@ -191,41 +192,48 @@ public void setSelectedClaimsProfileName(String selectedClaimsProfileName) {
     }
 
     /**
-     * Transform a map into a list of its keys.
+     * Helper method for extracting a profiles.json sub-component.
+     *
+     * @param attributesIdentifier name of the sub-structure to return from profiles.json (guestClaims,
+     *                             systemClaims, or configs are valid values).
+     * @return the given attribute component for the selected profile, or null if no profile was selected.
      */
-    private List<String> createListOfKeysFromMap(Map<String, String> map) {
-        return map.keySet()
-                .stream()
-                .collect(Collectors.toList());
+    @Nullable
+    private Object getProfileAttributes(String attributesIdentifier) {
+        if (selectedClaimsProfileName != null && !selectedClaimsProfileName.equals(DEFAULT_NAME)) {
+            Map<String, Object> profile = (Map<String, Object>) profiles.get(
```

Comment:
Root cause would be an invalid `profiles.json` document in `~/etc/ws-security` and will be caught in `ConfigurationAdmin`.